src/Dialogs/Plane/PlanePolarDialog.cpp

```diff
@@ -187,8 +189,9 @@ PlanePolarWidget::ListClicked()
 
   const PolarStore::Item &item = PolarStore::GetItem(list[result].int_value);
 
-  plane.reference_mass = item.reference_mass;
-  plane.dry_mass = item.reference_mass;
+  plane.polar_shape.reference_mass = item.reference_mass;
+  auto crew_mass_estimate = StaticString<32>(item.name).Contains(_T("PAS)")) ? 180. : 90.;
```

Comment:
I don't like matching a human-readable name for such "magic" sub strings. This is error prone. Never parse human-readable strings, please.
Also calling `StaticString` creates a truncated copy on the stack, and I don't know why you did that.

Reply:
See below.